pkgs/shells/powershell/default.nix

```diff
@@ -31,7 +32,7 @@ stdenv.mkDerivation rec {
     mkdir -p $out/share/powershell
     cp -r * $out/share/powershell
     rm $out/share/powershell/DELETE_ME_TO_DISABLE_CONSOLEHOST_TELEMETRY
-    makeWrapper $out/share/powershell/pwsh $out/bin/pwsh --prefix ${platformLdLibraryPath} : "${stdenv.lib.makeLibraryPath [ libunwind libuuid icu openssl curl ]}" \
+    makeWrapper $out/share/powershell/pwsh $out/bin/pwsh --prefix ${platformLdLibraryPath} : "${stdenv.lib.makeLibraryPath libraries}" \
```

Comment:
Are the packages opened with dlopen? platformLdLibraryPath is probably only required on macOS, where there is no patchelf.

Reply:
Yes, they are not explicitly linked to this libraries, that's why this approach was used in the first place.

This patch doesn't change that, just adds more libraries that /for some reason/ weren't an issue before.